coruscant/style/theme.js

```diff
@@ -18,6 +18,14 @@ const theme = {
     }
   },
 
+  spinnerContainer: {
```

Comment:
Do you plan on using the spinner without the defined spinnerContainer? I propose we just have Spinner.js export the spinner inside this container. Otherwise I'll make a BasicSpinner.js component that wraps a spinner in this defined style.

Reply:
Ya, you can have the Spinner wrapped in a container beforehand and then export it.